src/App.tsx

```diff
@@ -15,36 +16,90 @@
   'Garlic',
 ];
 
+const SORT_FILD_ALPHABETICCALY = 'alphabeticcaly';
+const SORT_FILD_LENGTH = 'length';
```

Comment:
There is a typo in the constant name `SORT_FILD_LENGTH`. It should be `SORT_FIELD_LENGTH` to maintain consistency and clarity.